src/main/java/com/github/princesslana/smalld/SmallD.java

```diff
@@ -236,11 +238,38 @@ public void sendGatewayPayload(String text) {
    * @throws HttpException.ClientException if there was a HTTP 4xx response
    * @throws HttpException.ServerException is there was a HTTP 5xx response
    * @throws HttpException for any non 2xx/4xx/5xx ressponse
+   * @throws IllegalArgumentException when the given path is malformed
    */
   public String get(String path) {
     LOG.debug("HTTP GET {}", path);
 
-    return http.send(path, Request.Builder::get);
+    return get(path, Collections.emptyMap());
+  }
+
+  /**
+   * Make a HTTP GET request to a Discord REST endpoint.
+   *
+   * <p>The path provided should start with {@code /} and will be appended to the base URL that has
+   * been configured.
+   *
+   * <p>This get request provides should provide a set of query parameters where the {@code Object}
+   * is a {@link java.lang.String} or can be transformed into a {@link java.lang.String} with {@link
+   * String#valueOf(Object)}.
+   *
+   * @param path the path to make the request to
+   * @param parameters the query string parameters
+   * @return the body of the HTTP response
+   * @throws com.github.princesslana.smalld.ratelimit.RateLimitException if the request was rate
+   *     limited
+   * @throws HttpException.ClientException if there was a HTTP 4xx response
+   * @throws HttpException.ServerException is there was a HTTP 5xx response
+   * @throws HttpException for any non 2xx/4xx/5xx ressponse
+   * @throws IllegalArgumentException when the given path is malformed
+   */
+  public String get(String path, Map<String, Object> parameters) {
+    LOG.debug("HTTP GET {}", path);
```

Comment:
We log the payload, we should log parameters, since this is for debugging what was actually sent.